src/dbcp/etl.py

```diff
@@ -282,18 +289,106 @@ def etl():
         "ballot_ready": etl_ballot_ready,
     }
     run_etl(etl_funcs, "data_warehouse")
-
     # Run private ETL functions
     etl_funcs = {
         "acp_projects": etl_acp_projects,
         "fyi_queue": etl_fyi_queue,
     }
     run_etl(etl_funcs, "private_data_warehouse")
 
-    logger.info("Sucessfully finished ETL.")
 
+def create_data_mart(engine, private_only: bool = False):  # noqa: C901
```

Comment:
I moved all this data mart table creation and coordination to etl.py and out of the data mart `__init__.py` to mimic the structure of the data warehouse.